sdk/resourcemanager/Azure.ResourceManager/src/Common/Generated/Models/KeyVaultProperties.cs

```diff
@@ -5,24 +5,19 @@
 
 #nullable disable
 
-using Azure.Core;
-
 namespace Azure.ResourceManager.Models
 {
     /// <summary> The KeyVaultProperties. </summary>
-    [PropertyReferenceType]
```

Comment:
Lets leave these attributes in place and add a provision in autorest.csharp to ignore reference types marked with Obsolete